include/mbedtls/platform_util.h

```diff
@@ -164,6 +164,14 @@ MBEDTLS_DEPRECATED typedef int mbedtls_deprecated_numeric_constant_t;
  */
 void mbedtls_platform_zeroize( void *buf, size_t len );
 
+void mbedtls_platform_memset( void *ptr, int value, size_t num);
+
+void mbedtls_platform_memcpy( void *dst, const void *src, size_t num );
```

Comment:
Please add doxygen description for all methods

Reply:
Will be done in a later commit, this PR is under development.